addons/website/static/src/systray_items/website_switcher.xml

```diff
@@ -13,10 +13,16 @@
         </t>
         <t t-foreach="getElements()" t-as="element" t-key="element_index">
             <DropdownItem
-                t-esc="element.name"
                 onSelected="element.callback"
-                class="element.class"
-            />
+                class="element.class">
+                <t t-if="element.noDomain">
+                    <span class="fa fa-warning me-2 text-warning"
```

Comment:
Can you move the tooltip to the whole dropdown item? It looks a bit weird/glitchy on the icon only, on top of being harder to notice/trigger (it's not obvious that there is a tooltip on it)
It would be better on the whole dropdown item so people will see it whatever they do or know
(Saying this because the triangle is so small that I actually kept closing it by moving 1-2px my mouse without doing it on purpose)
Ignore if too complicated

Reply:
It should be better now! 👍🏻